### PR TITLE
Add all tables

### PR DESCRIPTION
**Pull request by AdrianJendo**
Screenshot 2024-02-17 at 4 03 05 PM

### CODE REVIEW

File: backend/models/key.go
Comment (ABarroso647) [inline backend/models/key.go:1]: Why does this both have a userId and a user?

Comment (AdrianJendo) [reply backend/models/key.go:1]: I just followed this doc https://gorm.io/docs/belongs_to.html

**Review by ABarroso647 — DISMISSED** [review]
I would like config db to be separated line by line but that can be fixed in future small enuf Ill still approve

File: backend/models/generic.go [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
-	"gorm.io/gorm"
 	"time"
 )
 
 type Common struct {
-	ID        uint           `gorm:"primaryKey" json:"id"`
-	CreatedAt time.Time      `json:"created_at"`
-	UpdatedAt time.Time      `json:"updated_at"`
-	DeletedAt gorm.DeletedAt `gorm:"index" json:"deleted_at"`
+	Id        uint      `json:"id"`
```
Comment (ABarroso647) [inline backend/models/generic.go:8]: why is id no longer a primary key?

Comment (AdrianJendo) [reply backend/models/generic.go:8]: It still is, you don't need to specify `gorm:"primaryKey"` if the field name is Id: https://gorm.io/docs/models.html#Conventions. You can also see in the screenshot I posted for the descrition.

Comment (ABarroso647) [reply backend/models/generic.go:8]: figured was case sensitive, makes sense

**Review by ABarroso647 — DISMISSED** [review]

**Review by cameronkinsella — APPROVED** [review]

**Review by cameronkinsella — APPROVED** [review]

**Pull request merged**